examples/blog/components/BlogAuthor.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx h */
+import { h } from "preact";
+import { tw } from "@twind";
+
+export default function BlogAuthor({
+  authorName,
+  authorAvatar,
+}: {
+  authorAvatar: string;
+  authorName: string;
+}) {
+  return (
+    <div className={tw`flex items-center space-x-3`}>
```

Comment:
In Preact `class=` is preferred over `className=` (although both work). However `class` is being used on the Fresh documentation Web site so is probably preferred here too.